src/neo4j_genai/filters.py

```diff
@@ -90,7 +90,7 @@ class GteOperator(Operator):
 class InOperator(Operator):
     CYPHER_OPERATOR = "IN"
 
-    def cleaned_value(self, value):
+    def cleaned_value(self, value: Any) -> Any:
```

Comment:
I think value should be of type `list[Union[str, int, float]]` since we do this type check below anyways

Reply:
You mean a `Union[str, int, float]`

Reply:
5ab1e8ca0179137388dc7b8386e5a7cbc0008d06